capstone/src/main/webapp/dashboard-loader.js

```diff
@@ -31,13 +31,21 @@ function getClassrooms() {
     });
 }
 
+// TODO: Replace dummy values with real values in future PR
 function createClassroomDivElement(classroom) {
     let domparser = new DOMParser();
     let doc = domparser.parseFromString(`
-            <div class="classroom">
-                <h2>${classroom.entity.propertyMap.subject}</h2>
-                <p><a href="form.html">Classroom Link</a></p>
+            <div class="card margin margin-left">
+                <img class="card-img-top" src="/assets/soundwave.svg" alt="Room Card">
+                <div class="card-body text-center">
+                    <h5 class="card-title">${classroom.entity.propertyMap.subject}</h5>
```

Comment:
I can't seem to find the shadows of the cards. They look pretty sick though. Are they in the CSS?

Reply:
Also cards are formatted so that if they have extra text, they will become larger as a whole. Just letting you know but since this is UI, no need to change IMO.

Reply:
![Screenshot 2020-08-13 at 8 46 25 PM](https://user-images.githubusercontent.com/32047306/90211694-4c24a100-dda6-11ea-9da4-58aea6d302c0.png)

Reply:
Yea I knew that. If we want to make it a set size let me know

Reply:
and the shadow is in the css under "card:hover"